plugins/reverse-proxy.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: reverse-proxy
+spec:
+  version: v0.2.2
+  homepage: https://github.com/rajatjindal/kubectl-reverse-proxy
+  shortDescription: Starts reverse proxy for the k8s service pods.
+  description: |
+    Runs the reverse proxy locally and route traffic to all pods behind the given service object. 
+    
+    It is different from the existing proxy plugins in that it starts a reverse proxy locally (instead of e.g. starting a socat proxy pod in the cluster) and 
+    updates the backend config of that reverse proxy (currently using caddy) by doing port-forwarding and adding those listeners as backends.
```

Comment:
recommended line wrapping at 80 chars (you can try installing locally from manifest and seeing how it renders)

Reply:
done. thank you.